packages/react-components/src/input/src/input.jsx

```diff
@@ -96,6 +95,11 @@ function throwWhenMutuallyExclusivePropsAreProvided({ button, icon, iconPosition
 
 export function PureInput(props) {
     const { autofocus, autofocusDelay, className, fluid, icon, iconPosition, button, size, loading, disabled, children, forwardedRef, ...rest } = props;
+    const SIZES_TO_BUTTON = {
+        [SMALL]: "tiny",
+        [MEDIUM]: "small",
+        [LARGE]: "medium"
```

Comment:
Instead of hardcoding "tiny", "small", "medium" you should also used constant from `"../../shared`